src/samVisualize/graphGeneration/cfnTemplateGraphGenerator.ts

```diff
@@ -143,6 +146,7 @@ export function generateGraphFromYaml(inputYaml: string): GraphObject | undefine
         for (const resourceName of Object.keys(resources)) {
             // If a resource does not contain an object body, it cannot be defined in the graph
             if (!_.isObjectLike(resources[resourceName])) {
+                getLogger().error('SAM Visualize: Error rendering SAM template. One or more resources is invalid.')
```

Comment:
Provide a list of affected `resourceName`s?

Reply:
Saying one or more is not concise, I've changed it to say "The `specifc resource name` resource definition is invalid"